app/flow_io/decorators/spree_variant_decorator.rb

```diff
@@ -20,7 +20,7 @@ def self.flow_truncate
 
   # syncs product variant with flow
   def flow_sync_product
-    # flow_client.items.put_by_number flow_org, sku, flow_item
+    return unless product.images.first
```

Comment:
this was a fix when bulk inserting products to the database.

all products we have an image, but it is possible that images are uploaded in another thread so they are not available when requested.

prices will be right and catalogs can be re-synced to get fresh images in the console, so I will fix to not request an image for sync.